src/tags/include_tag.rs

```diff
@@ -0,0 +1,66 @@
+
+use Renderable;
+use context::Context;
+use token::Token;
+use LiquidOptions;
+use error::{Result, Error};
+
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+struct Include {
+    partial_path: String,
+}
+
+impl Renderable for Include {
+    fn render(&self, _context: &mut Context) -> Result<Option<String>> {
+        let path_str = self.partial_path.clone();
+        let path = Path::new(&path_str);
+
+        // check if file exists
+        if !path.exists() {
+            return Error::renderer(&format!("{:?} does not exist", path));
+        }
+
+        let mut file = match File::open(path) {
+            Ok(file) => file,
+            Err(e) => return Error::renderer(&format!("[std::io::Error] {}", e)),
+        };
+
+        let mut content = String::new();
+        match file.read_to_string(&mut content) {
+            Ok(_) => Ok(Some(content)),
+            Err(e) => Error::renderer(&format!("[std::io::Error] {}", e)),
+        }
+    }
+}
+
+pub fn include_tag(_tag_name: &str,
+                   arguments: &[Token],
+                   _options: &LiquidOptions)
+                   -> Result<Box<Renderable>> {
+    let mut args = arguments.iter();
+
+    match args.next() {
+        Some(&Token::Identifier(ref path)) => Ok(Box::new(Include { partial_path: path.clone() })),
+        arg => Error::parser("Token::Identifier", arg),
+    }
+}
+
+#[cfg(test)]
+mod test {
+    use context::Context;
+    use Renderable;
+    use parse;
+
+    #[test]
+    fn include_tag() {
+        let text = "{% include tests/fixtures/additional/default.liquid %}";
```

Comment:
I would try not to depend on an external file being in the right place at the right time for a unit test - I've been bitten by that too often. If you can't fake the file access (e.g. read from a file-like memory buffer or something), and in this case I think the test really _does_ need to hit the filesystem, I usually:

1. Create a temp directory at the start of the test,
2. Create the file with the content I want to test in that directory
3. Point whatever code that needs the file at the temp file
4. Destroy the temp dir on test exit

Have a peek at the[ `tempdir`](https://doc.rust-lang.org/tempdir/tempdir/index.html) crate, which gives you an RAII scope-managed temp directory, which would probably do a lot of the heavy lifting for you.

(...but I'll defer to @johannhof if he thinks I'm being too pedantic)